go/worker/storage/committee/node.go

```diff
@@ -360,6 +363,32 @@ func (n *Node) updateExternalServicePolicyLocked(snapshot *committee.EpochSnapsh
 	// Update storage gRPC access policy for the current runtime.
 	n.grpcPolicy.SetAccessPolicy(policy, n.commonNode.Runtime.ID())
 	n.logger.Debug("set new storage gRPC access policy", "policy", policy)
+
+	// Notify the sentry nodes of the new policy.
+	for _, addr := range n.workerCommonCfg.SentryAddresses {
```

Comment:
See above, with this centralized it shouldn't be needed at all as calling `SetAcceessPolicy` above will propagate the policy.